### DISCUSSION TITLE
GMTSAR config file

### DISCUSSION THREAD

**Discussion by ditafaith** [insarlab/MintPy#1078] [Q&A]
I didn't find a sample for GMTSAR in following address. 
https://github.com/insarlab/MintPy/blob/main/docs/templates/README.md

How I can produce a "config" txt file to process stack of ifgs from GMTSAR ?

**Comment by yunjunz** [comment]
Different InSAR processors have slightly different template options in the config file for the `load_data` step only, which is described in https://mintpy.readthedocs.io/en/latest/dir_structure/; the rest of the options are the same. You can find the full template option by running `smallbaselineApp.py -H`, and copy any of them into your "custom config" txt file.

**Comment by ditafaith** [comment]
> Different InSAR processors have slightly different template options in the config file for the `load_data` step only, which is described in https://mintpy.readthedocs.io/en/latest/dir_structure/; the rest of the options are the same. You can find the full template option by running `smallbaselineApp.py -H`, and copy any of them into your "custom config" txt file.

I was confused to understand. Actually all directories have followed 
https://mintpy.readthedocs.io/en/latest/dir_structure/#gmtsar, then I copied all containing smallbaselineApp.py -H to config_template.txt, as attached. 
[config_template.txt](https://github.com/insarlab/MintPy/files/12437565/config_template.txt)

Then, run command smallbaselineApp.py config_template.txt under mintpy directory, but it is not work at all.
Which part I should customized ? and what is the minimum interferogram for mintpy can process SBAS ?
I am sorry, since there is a no sample how to use MintPy using GMTSAR
https://mintpy.readthedocs.io/en/latest/demo_dataset/

I was thinking, the file is containing like this 

## manually specify the following attributes since they are missing from gmtsar products
HEADING         = -168.0     #[float], satellite heading angle, measured from the north in clockwise as positive
                             # One could open the *.kml file in Google Earth and measure it manually
ORBIT_DIRECTION = DESCENDING #[ASCENDING, DESCENDING]

mintpy.load.processor   = gmtsar
##---------interferogram datasets:
mintpy.load.unwFile     = $DATA_DIR/StHelensEnvDT156/interferograms/*/unwrap_ll.grd
mintpy.load.corFile     = $DATA_DIR/StHelensEnvDT156/interferograms/*/corr_ll.grd
##---------geometry datasets:
mintpy.load.demFile     = $DATA_DIR/StHelensEnvDT156/geometry/topo_ll.grd
